app/features/hit_fox_feed.rb

```diff
@@ -0,0 +1,33 @@
+require 'github_feed'
+require 'yajl'
+
+class HitFoxFeed
+  def initialize(user, org, token)
+    @feed = GithubFeed.new(user,org, token)
+    @redis = $redis
```

Comment:
i use the inizializer approach so that i can conf to use different db depending on the environment.

For example 
development:
  url: redis://127.0.0.1:6379/1
test:
  url: redis://127.0.0.1:6379/2